.rubocop.yml

```diff
@@ -15,6 +15,8 @@ Bundler/OrderedGems:
 # because we have code that is clearer this way
 Metrics/AbcSize:
   Max: 16
+  Exclude:
+    - 'lib/sparql_to_sw_solr/instance_title_fields.rb'
```

Comment:
will we need this if we adjust the method per suggestion below?  (I don't mind it, but if we don't need it ...)